TestsExample/App.js

```diff
@@ -98,6 +98,7 @@ import Test1981 from './src/Test1981';
 import Test2008 from './src/Test2008';
 import Test2048 from './src/Test2048';
 import Test2069 from './src/Test2069';
+import Test2118 from './src/Test2118';
```

Comment:
Do we need this test in paper tests, if it's only a problem with Fabric?

Reply:
This will be handled here: https://github.com/software-mansion/react-native-screens/pull/2135